library/PluginManager.cpp

```diff
@@ -1019,7 +1012,7 @@ void PluginManager::OnStateChange(color_ostream &out, state_change_event event)
 
 void PluginManager::registerCommands( Plugin * p )
 {
-    cmdlist_mutex->lock();
+    lock_guard<std::mutex> lock{*cmdlist_mutex};
```

Comment:
Need to remove `cmdlist_mutex->unlock();` at the end of the function, otherwise it double-unlocks (and causes a deadlock later).